Dockerfile

```diff
@@ -24,14 +24,15 @@ RUN go build -ldflags="-s -w" -a -o /TechChallengeApp
 RUN swagger generate spec -o /swagger.json
 
 FROM alpine:latest
-
 WORKDIR /TechChallengeApp
 
 COPY assets ./assets
-COPY conf.toml ./conf.toml
+COPY conf.toml /opt/conf.toml
```

Comment:
Updating the location to make it usable via mount option from host file. Thus each host can have individual setup. e.g. the same image can be used in dev, test environment using separate conf.toml mount from host.